app/models/ems_refresh/save_inventory.rb

```diff
@@ -105,7 +107,7 @@ def save_vms_inventory(ems, hashes, target = nil)
             h[:location] = "unknown" if h[:location].blank?
 
             # build a type-specific vm or template
-            found = ems.vms_and_templates.build(h)
+            found = ems.vms_and_templates.klass.new(h)
```

Comment:
Good find on this one. But it is so curious.

Do you think this is a 5.0.1 regression type thing, or the way to move forward?